Source/ConversationV1/Conversation.swift

```diff
@@ -1266,19 +1264,19 @@ public class Conversation {
 
      - parameter workspaceID: The workspace ID.
      - parameter entity: The name of the entity.
-     - parameter properties: A CreateValue object defining the content of the new value for the entity.
+     - parameter body: A CreateValue object defining the content of the new value for the entity.
```

Comment:
Looks like there are some changes in the `staging/conversation` branch that haven't been merged to `master`. Here's a [diff][diff] to compare them. One of the changes is an `x-alternate-name` of `properties` for this body parameter. So that's why the name differs here.

[diff]: https://github.ibm.com/Watson/developer-cloud--api-explorer/compare/master...staging/conversation